README.md

```diff
@@ -1 +1,6 @@
 # curriculum-cr-training-mobile-menu
+
+
+This repo host an initial code for the tasks performed during the Code Reviewers training.
+
+Application is based on this [Figma design](https://www.figma.com/file/t3EJUCAEViw3QasuJLPLVT/Microverse-Student-Potfolio-Templates-Main?node-id=1%3A1471).
```

Comment:
The most important rule in maintaining a professional repo is to always think about people that you share your code. A descriptive readme file is very important in making your project look professional.
Please use this [template ](https://github.com/microverseinc/readme-template) to update your readme file, However, please remember to customize it to your project.

Reply:
Thank you. Done